Created a "Past Sponsors" section on the website

rkd-sh64 opened the pull request:
I have made the necessary changes to show past sponsors under one section without categorizing them.This is for issue #18 
I have modified PartnersAndSponsorsSection.jsx file to show the Past Sponsors and did not create a seperate file for that.
If any further changes are required, do let me know.

It looks like this:
![Past Sponsors](https://github.com/hack4bengal/s3/assets/132741672/355f5e43-2945-4cdc-abfd-a91a32bce8b5)

rkd-sh64 committed:
Created the image section to show the past sponsors

rkd-sh64 committed:
Adding the required styles for past sponsors

rkd-sh64 committed:
Adding the required images of past sponsors

rkd-sh64 committed:
Created PastSponsors.js to store the data of the images in a single file

rkd-sh64 committed:
Updated styles

tamalCodes reviewed (changes requested):
Hi @rohits-web03 , i think it would be better if we could use a marque format or a carousel because currently the way it is, it would take a lot of screen space.

Plus some images looks very stretched, please fix those too.

tamalCodes commented on src/sections/PartnersAndSponsorsSection/PartnersAndSponsorsSection.jsx:
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import sponsers from "../../assets/data/sponsersData";
+import {gov,sponsors} from "../../assets/data/PastSponsors";
```
And also instead of adding Two different data structures, why not just add the gov pictures in the sponsor's data itself? It would avoid confusion for the time being.

tamalCodes commented on src/sections/PartnersAndSponsorsSection/PartnersAndSponsorsSection.jsx:
```diff
@@ -21,10 +22,28 @@ const PartnersAndSponsorsSection = () => {
           <hr />
-          <h3 style={{ textAlign: "center" }}>To be decided soon...</h3>
+          {/* <h3 style={{ textAlign: "center" }}>To be decided soon...</h3> */}
+          <div className="sponsors-img">
```
Also please remove this line, instead of commenting it out.

tamalCodes commented on src/sections/PartnersAndSponsorsSection/PartnersAndSponsorsSection.css:
```diff
@@ -142,6 +142,47 @@
+  margin-top: 1rem;
+}
+
+#img-other{
+  display: flex;
+  gap: 4rem;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+#img-other a div{
+  width: 13rem;
+  height: 8rem;
+  object-fit: contain;
+}
+
+#img-other a div img{
+  width: 100%;
+  height: 100%;
+}
+
+#img-gov img{
+  width: 15rem;
+  height: 15rem;
+}
+
+#img-gov{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
```
Would be better if you could use classnames for stylings.

tamalCodes commented on src/assets/data/PastSponsors.js:
Some pictures might need custom width and height, first try a marque format or a carousel format which autoplays, and then if needed we can add another field which satisfies the same.

rkd-sh64 commented:
@tamalCodes Okay👍
I will try to make these changes.

rkd-sh64 committed:
Using a single array named sponsors to export all the images istead of two

rkd-sh64 committed:
Added the image slider and removed unnecessary lines of code

rkd-sh64 committed:
Merge branch 'main' of https://github.com/rohits-web03/Hack4Bengal-s3 into past-sponsors

rkd-sh64 commented:
@tamalCodes I have made the changes

- added image carousel, 
- removed unnecessary lines of code from PartnersAndSponsorsSection.jsx, 
- using classes(not ids) for styling, and 
- using one array named 'sponsors' to export all images from PastSponsors.js

If any further changes are required, do let me know.

Also there are a lot of styles in PartnersAndSponsorsSection.css which are not used currently, Should I remove them?
